scarlet/lite/models.py

```diff
@@ -0,0 +1,607 @@
+# This module contains classes and methods to use the so-called "lite" version of scarlet.
+# In this instance "lite" just means scarlet stripped down to the bare essentials needed
+# to deblend observations from a single instrument, where all of the observations have
+# been resampled to the same pixel grid (but may have differing PSFs).
+# Because some of the assumptions and constraints are different, this module is designed
+# to run faster and use less memory than the standard scarlet modules while obtaining
+# a similar log likelihood.
+import numpy as np
+
+from scarlet.lite.measure import weight_sources
+
+from ..bbox import overlapped_slices, Box
+from ..renderer import convolve
+from .. import fft, interpolation, initialization
+from ..constraint import  MonotonicityConstraint
+from .utils import insert_image
+
+
+class LiteComponent:
+    """A base component in scarlet lite
+
+    If `sed` and `morph` are arrays and not `LiteParameter`s then the
+    component is not `initialized` and must still be initialized by
+    another function.
+    """
+    def __init__(self, center, bbox, sed=None, morph=None, initialized=False,
```

Comment:
This is a base class, but it assumes sed and morphology parameterization just like `LiteFactorizedComponent`. I'm not sure why these two classes are implemented separately.

Reply:
I added to the docsting to `LiteComponent`. Basically, a `LiteComponent` has an SED and morphology but those are not yet initialized, meaning they are numpy arrays. But in `LiteFactorizedComponent` they are `LiteParameter`s. This allows the initialization and parameterization to be modular, so that initialization happens first, followed by parameterization.